vignettes/adtte.Rmd

```diff
@@ -205,6 +204,63 @@ We advise you consult the `{admiral}`
 ["Creating a BDS Time-to-Event ADaM" vignette](https://pharmaverse.github.io/admiral/articles/bds_tte.html)
 for further guidance on the different options available and more examples.
 
+One extra common oncology case we include here is around PFS when censoring
+at next anti-cancer therapy. This could either be controlled using `ANLzzFL` as
+explained in the ADRS vignette, so that records after next anti-cancer therapy
+never contribute to the PD and DEATH parameters. Or alternatively you can control
+this on the ADTTE side by filtering which records are used in
+`admiral::event_source()` and `admiral::censor_source()`.
+The censor could be set as whichever date your analysis requires,
+e.g. date of last tumor assessment prior to next anti-cancer therapy or last
+radiological assessment.
+This case is demonstrated in the below example (where `NACTDT` would be
+pre-derived as first date of next anti-cancer therapy and `LASTANDT` as date of
+last assessment prior to next anti-cancer therapy).
+
+```{r, eval=FALSE}
+pd_nact_event <- event_source(
+  dataset_name = "adsl",
+  filter = PDDT < NACTDT,
```

Comment:
I think this filter condition is not work as we intended and can confuse. The reason is it will remove the PDs if the patient has not got any new anti-cancer therapy.  I suggest (!is.na(PDDT) & !is.na(NACTDT) & PDDT < NACTDT) | is.na(PDDT)==FALSE.
Same suggestion for death_nact_event below..

Reply:
Good spot! This one can be simplified but will definitely update to ensure we cover missing NACTDT case. R doesn't work like SAS when comparing < or > for potential missings as FYI.